schedule-ui.xml

```diff
@@ -140,7 +140,7 @@
                 <area>0,0,285,111</area>
                 <state name="active">
                     <shape name="buttonbackground">
-                        <area>0,0,285,84</area>
+                        <area>0,0,285,83</area>
```

Comment:
Are you not full screen 1920x1080?  I get all 7 icons.  But 83 here also works for me so we could change this.